src/main/java/team/reborn/energy/Energy.java

```diff
@@ -6,6 +6,10 @@
 import java.util.function.Function;
 import java.util.function.Predicate;
 
+/**
+ * @deprecated Use the new energy API instead. (Under the team/reborn/energy/api package).
+ */
+@Deprecated(forRemoval = true)
```

Comment:
This is fine, would be nice to remove them from the source tree, I can possibly do some gradle stuff to shade this in for now, or just move to another source set.

Reply:
Not sure what the best approach is... I don't want to complicate the build logic too much as I don't really understand it.